T/TinyXML/build_tarballs.jl

```diff
@@ -0,0 +1,46 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "TinyXML"
+version = v"2.6.2"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://downloads.sourceforge.net/project/tinyxml/tinyxml/$(version)/tinyxml_$(version.major)_$(version.minor)_$(version.patch).tar.gz",
+                  "15bdfdcec58a7da30adc87ac2b078e4417dbe5392f3afb719f9ba6d062645593"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/tinyxml
+
+c++ -c -Wall -Wno-unknown-pragmas -Wno-format -O3 tinyxml.cpp -o tinyxml.o
+c++ -c -Wall -Wno-unknown-pragmas -Wno-format -O3 tinyxmlparser.cpp -o tinyxmlparser.o
+c++ -c -Wall -Wno-unknown-pragmas -Wno-format -O3 xmltest.cpp -o xmltest.o
+c++ -c -Wall -Wno-unknown-pragmas -Wno-format -O3 tinyxmlerror.cpp -o tinyxmlerror.o
+c++ -c -Wall -Wno-unknown-pragmas -Wno-format -O3 tinystr.cpp -o tinystr.o
+mkdir -p "${bindir}"
+c++ -o "${bindir}/xmltest${exeext}" tinyxml.o tinyxmlparser.o xmltest.o tinyxmlerror.o tinystr.o  
+
+install_license readme.txt
+
+mkdir -p "${includedir}"
+cp *.h "${includedir}/."
```

Comment:
Why distributing the header files?  This isn't a library

Reply:
Can this be made a library? This was my best effort at reading the Makefile and replicates what is produced locally for me.

This would be in support of a potential future [OTB](https://www.orfeo-toolbox.org/CookBook/CompilingOTBFromSource.html) build. See [this script](https://gitlab.orfeo-toolbox.org/orfeotoolbox/otb/-/blob/develop/SuperBuild/patches/TINYXML/CMakeLists.txt) for how they use it.

Reply:
I think it should be ok now?